spec/fixtures/files/conferences/big_blue_button_get_recordings_one.xml

```diff
@@ -0,0 +1,39 @@
+<response>
+  <returncode>SUCCESS</returncode>
+  <recordings>
+    <recording>
+      <recordID>0225ccf234655ae60658ccac1e272d48781b491c-1511812307014</recordID>
+      <meetingID>instructure_web_conference_3Fn2k10wu0jK7diwJHs2FkDU0oXyX1ErUZCavikc</meetingID>
+      <internalMeetingID>0225ccf234655ae60658ccac1e272d48781b491c-1511812307014</internalMeetingID>
+      <name>Conferencia de Development 101</name>
+      <isBreakout>false</isBreakout>
+      <published>true</published>
+      <state>published</state>
+      <startTime>1511812307014</startTime>
+      <endTime>1511812327102</endTime>
+      <participants>1</participants>
+      <metadata>
+        <meetingName>Jesus Federico's conference</meetingName>
+        <meetingId>instructure_web_conference_3Fn2k10wu0jK7diwJHs2FkDU0oXyX1ErUZCavikc</meetingId>
+        <canvas-recording-ready-user>John Administrator &lt;admin@123it.ca&gt;</canvas-recording-ready-user>
+        <canvas-recording-ready-url>http://canvas.instructure.com/api/v1/courses/1/conferences/8/recording_ready</canvas-recording-ready-url>
+        <isBreakout>false</isBreakout>
+      </metadata>
+      <playback>
+        <format>
+          <type>presentation</type>
+          <url>http://bbb.instructure.com/playback/presentation/2.0/playback.html?meetingId=0225ccf234655ae60658ccac1e272d48781b491c-1511812307014</url>
+          <processingTime>9922</processingTime>
+          <length>0</length>
+          <preview>
+            <images>
+              <image alt="Welcome to" height="136" width="176">http://bbb.instructure.com/presentation/0225ccf234655ae60658ccac1e272d48781b491c-1511812307014/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1511812307214/thumbnails/thumb-1.png</image>
+              <image alt="(this slide left blank for use as a whiteboard)" height="136" width="176">http://bbb.instructure.com/presentation/0225ccf234655ae60658ccac1e272d48781b491c-1511812307014/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1511812307214/thumbnails/thumb-2.png</image>
+              <image alt="(this slide left blank for use as a whiteboard)" height="136" width="176">http://bbb.instructure.com/presentation/0225ccf234655ae60658ccac1e272d48781b491c-1511812307014/presentation/d2d9a672040fbde2a47a10bf6c37b6a4b5ae187f-1511812307214/thumbnails/thumb-3.png</image>
+            </images>
+          </preview>
+        </format>
+      </playback>
+    </recording>
+  </recordings>
+</response>
```

Comment:
Lets make sure all the urls in here are not real urls

Reply:
Done